package.json

```
@@ -1,16 +1,21 @@
 {
   "name": "@storybook-community/storybook-addon-mock-root",
   "description": "A monorepo for storybook-addon-mock",
+  "version": "10.1.2",
   "private": true,
   "workspaces": [
     "packages/*"
   ],
   "scripts": {
     "clean": "rm -rf dist",
     "build": "yarn run addon:build && yarn run docs:build",
+    "build:storybook": "yarn workspace @espoc/storybook-addon-mock-docs run build",
```

```suggestion
    "build:storybook": "yarn workspace @storybook-community/storybook-addon-mock-docs run build",
```

Old habits die hard :)